components/brave_rewards/resources/rewards_panel/lib/extension_host.ts

```diff
@@ -368,8 +368,6 @@ export function createHost (): Host {
     // the data that we have, rather than a stalled loading indicator.
     setTimeout(() => { stateManager.update({ loading: false }) }, 3000)
 
-    await proxy.handler.startRewards()
-
     await loadPanelData()
```

Comment:
I think we'll need to reload all panel data when Rewards is "initialized" due to opt-in. We might be able to do this by adding a call to `loadPanelData` inside of the `onRewardsEnabled` handler in this file.

Reply:
Updated in 3342b11fc9fe6156072e92080eefbc7ca0ada8f9